pyproject.toml

```diff
@@ -28,7 +28,7 @@ requires-python = ">=3.8"
 dependencies = [
     "click>=8.1.3",
     "ruyaml>=0.91.0",
-    "maison>=1.4.0",
+    "maison>=1.4.0,<1.4.3",
```

Comment:
make sure we don't use neither `maison 2.0.0` (breaks interface) nor `maison 1.4.3` (which updates to `pydantic 2.X` and causes other incompatibilities with other dependencies).

Reply:
It would have been ideal to have first released a version that pinned `maison` and supported Python 3.8 before subsequently releasing a version that dropped support for Python 3.8 with the latest `maison`. This would have allowed people who are still on Python 3.8 to pin their `yamlfix` version. As it stands, I don't think any existing version of `yamlfix` will work with Python 3.8 because of the `maison` issue.

Is this still possible?

Reply:
Hi @gandhis1, you are right, sorry for the inconveniences. I've created a new branch python3.8 and released version 1.16.1 with the fixed dependencies.